### DISCUSSION TITLE
Markdown edit page bug for RTL

### DISCUSSION THREAD

**Discussion by talalvas** [requarks/wiki#3644] [Error / Bug Report]
### Actual behavior
edit section on the the left side of the screen and the preview section moved to the right.
text is also aligned to the left
![image](https://user-images.githubusercontent.com/9365114/76525844-4a605580-6475-11ea-837e-f3bbefa67db6.png)

### Expected behavior
After 12.13.1 update, the edit section moved to the right side of the screen and the preview section moved to the left. and the text on bouth sections was aligned to the right, exactly as it should for RTL users.
The problem is that suddenly they swapped places as they were prior to the update.

### Steps to reproduce the behavior
I think that the action that triggered it was restarting of the server. but i'm not 100% sure.
another possible reason is adding my company logo but it sound very strange.
i couldn't find a way to undo the company logo addition. the only thing i managed to do is to remove the logo I've added but it didn't retrieved wiki.js logo.

**Comment by qpsamet** [comment]
Same here with Persian locale on Wiki.js version 2.2.51

**Comment by talalvas** [comment]
Hi, I've found out that the problem is with this div in the edit page:
`<div data-app="true" class="v-application editor v-application--is-LTR theme--light" id="app">`
the direction is **LTR** when it should be **RTL**.
I guess that the last update restored it to the previous settings
can you fix it in the next update and try to edit a page in RTL environment?
